android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -883,6 +883,14 @@ fun String.toMediaType(): MediaType? = when (this) {
     else -> null
 }
 
+data class MediaControlConfig(
+    var isEnabled: Boolean = false,
+)
```

Comment:
Shouldn't the default value be `true`?

Reply:
Address in 2ee1f112df2f06585a9716b6227c7de4168b1ee8